### ISSUE TITLE
Regression in #149, Packaging failing on Windows with Git Bash

### ISSUE THREAD

**Issue by frereit** [megastep/makeself#162]
OS: Windows 10
Bash: Git Bash

When using the repo from c16151f9db709a71ac799063b8ceb32c62443b91 this command works fine. You can see that `getdown-stub` contains the required file (`run.sh`) and also some other files I want to unpack to the pc where this is installed. 
```
> bash makeself.sh getdown-stub installer.sh Installer ./run.sh
Header is 587 lines long

About to compress 186 KB of data...
Adding files to archive named "installer.sh"...
a .
a ./getdown.jar
a ./getdown.txt
a ./run.sh
CRC: 1946495856
MD5: b71a655854ce42d10e7848d4be0576da

Self-extractable archive "installer.sh" successfully created.
```

However in d22a2dafbf6a10ddf45da9f8082929e29d46054e this does not work anymore: 
```
>bash makeself.sh getdown-stub installer.sh Installer ./run.sh
Header is 626 lines long

About to compress 186 KB of data...
Adding files to archive named "installer.sh"...
FIND: Parameterformat falsch
tar.exe: no files or directories specified
ERROR: failed to create temporary archive: /tmp/mkself10228.tar
```

The `getdown-stub` folder is identical in both cases.
If I can further help triage this issue, please let me know.

**Comment by megastep** [comment]
It sounds like the call to `find` is at fault here? I'm not familiar with the way Windows 10 handles these things.

**Comment by frereit** [comment]
Oh, I did not notice that there was an error in German (my windows language). It roughly translates to "wrong parameter format" so yeah it is a problem with the call to `find`. Curiously enough I cannot reproduce this on my home PC right now, even though I followed almost identical setup steps. I will investigate this further when I have access to my work PC again. I tried both the latest release, d22a2dafbf6a10ddf45da9f8082929e29d46054e (which did not work on my work PC) and c16151f9db709a71ac799063b8ceb32c62443b91 and all of them ran without any errors.
Oh well, that's just how debugging goes I guess ;). Thanks for the response.
